Client/src/Components/Label/index.jsx

```diff
@@ -119,13 +119,14 @@ ColoredLabel.propTypes = {
  * @param {Object} props
  * @param { 'up' | 'down' | 'cannot resolve'} props.status - The status for the label
  * @param {string} props.text - The text of the label
+ * @param {boolean} props.hasDot - Whether to show a dot or not
```

Comment:
Since I wasn't the one who added customStyles, I chose not to make any changes related to it. There might be a reason for its implementation that was discussed in a previous PR.